source/ui/REIXS/REIXSSampleChamberButtonPanel.cpp

```diff
@@ -59,10 +57,69 @@ REIXSSampleChamberButtonPanel::~REIXSSampleChamberButtonPanel()
 void REIXSSampleChamberButtonPanel::onStopButtonClicked()
 {
 	REIXSSampleChamber* chamber = REIXSBeamline::bl()->sampleChamber();
-	chamber->x()->stop();
-	chamber->y()->stop();
-	chamber->z()->stop();
-	chamber->r()->stop();
+	chamber->beamNormalTranslation()->stop();
+	chamber->beamHorizontalTranslation()->stop();
+	chamber->beamVerticalTranslation()->stop();
+	chamber->beamVerticalRotation()->stop();
```

Comment:
why don't we add a "stop()" function for `REIXSSampleChamber`, which will call the stop functions of those motors?